### ISSUE TITLE
URN UUID for person

### ISSUE THREAD

**Issue by ncarboni** [swiss-art-research-net/rdmi#56]
Present 20249 records of person identified by a urn uuid

**Comment by ncarboni** [comment]
UUID list here:
[actor-uuid.tsv.zip](https://github.com/swiss-art-research-net/rdmi/files/4276672/actor-uuid.tsv.zip)


@ymark @MinadakisNikos Do you know where they come from?

**Comment by MinadakisNikos** [comment]
This is not coming by the mappings, it should be an old issue that is fixed now. We will check for sure after the re-importing of the data.

**Comment by ncarboni** [comment]
👍 Gonna keep it open as a reminder for verification

**Comment by ncarboni** [comment]
I verified and it is coming by the mapping of Groups and places. 
In the case of groups It was creating urn for places when not present (because of the instance generator was URIorUUID with xpath). I added a 

```
<if>
	<exists>text()</exists>
</if>
```
in all the cases we had 

```
<instance_generator name="URIorUUID">
	<arg name="text" type="xpath">text()</arg>
</instance_generator>
```
In case of Places. it was was a similar problem but for identifier. 
Person was ok.

Everything is fixed now. I am going to re-upload them on github soon
